### DISCUSSION TITLE
Grouping databases in Grafana under the same instance

### DISCUSSION THREAD

**Discussion by Twinsenhub** [cybertec-postgresql/pgwatch2#821] [Ideas]
Would it be possible to modify the grouping and filtering in Grafana to filter instances and the associated databases? If we have for example more than 700 instances and each instance has 10 databases, then we have 7k rows for filtering and that is not very user friendly. 
So for each instance there would be only one row and after filtering there would be a list of databases to choose from. Like example pics in a attachment here.
![grafana1](https://github.com/user-attachments/assets/45cecc86-f984-417c-ab57-f2c91afa2bc7)
![grafana2](https://github.com/user-attachments/assets/d8cb3a44-208e-434a-bb4b-d29abac4107e)

Thank you for considering or for pgwatch3. 
Martin

**Comment by pashagolub** [comment]
Hello. Yes, it's absolutely possible! You can change your dashboards as you want! Don't forget to share back your solution! 👍
